all/projectAlcazar/include/system.h

```diff
@@ -42,9 +42,12 @@ typedef enum
 	ePerif_GPIOF = 22,
 
 	//These are bits from the RCC_APB2RSTR register!
-	ePerif_TIM1 = 11,
-	ePerif_TIM15 = 16,
-	ePerif_SPI1EN = 12,
+	ePerif_TIM1 = 11, //TIM1EN
+	ePerif_TIM15 = 16, //TIM15EN
+	ePerif_SPI1EN = 12,  //SPI1EN
+	//The bits from RCC_APB1ENR register
+	// TIM3EN
+	ePerif_TIM3EN = 1
```

Comment:
this bit is in the (RCC_APB1ENR) register.
You should also define that register and in enablePeripheral function in system.c you should set the bit in that register

Reply:
this is done in latest code uart code because the same situation happend with usart2en bit